docs/tsg_install.md

```diff
@@ -60,6 +60,27 @@ iscsiadm version 2.0-874
 
 To resolve this issue, do not install `open-iscsi / iscsi-initiator-utils` on the host nodes when using the Rancher Container Engine (RKE).
 
+## How can I do selective in disks for creating storage pool using cStor?
+
+With latest 0.7.0 OpenEBS release, following type of disk types/paths are excluded by NDM to identify the disks to create cStor Pool on nodes. 
+```
+loop,/dev/fd0,/dev/sr0,/dev/ram,/dev/dm-
+```
+You can also customize with adding more type of disks associated with your nodes like used disks,unwated disks etc. This has to be done in your downloaded 'openebs-operator-0.7.0.yaml' before the installation. You have to add the exact device path in openebs-ndm-config under ConfigMap in `openebs-operator.yaml` as follows.
```

Comment:
You can also customize by adding more disk types associated with your nodes. For example, used disks, unwanted disks and so on. This must be done in the 'openebs-operator-0.7.0.yaml' file that you downloaded before installation. Add the device path in openebs-ndm-config under ConfigMap in the 'openebs-operator.yaml` file as follows.